apps/src/music/views/SoundsPanel.tsx

```diff
@@ -38,6 +38,20 @@ const getLengthRepresentation = (length: number) => {
   return lengthToSymbol[length] || length;
 };
 
+const decimalToFraction = (decimal: number): string => {
```

Comment:
This is slick 🙂 

FWIW, I think we only ever have values of 1/2 or 1/4 as fractional sound lengths (right @breville?). We also do have an existing function in this file, `getLengthRepresentation` that converts the decimal value to the fractional unicode value. I wonder how the screen reader would readout if we just fed it the unicode value (i.e. `\u00bd` or `\u00bc`)? Alternatively, when we get the unicode value, we can also explicitly get the fraction value as a string (i.e. `"1/2"`, `"1/4"`)

Reply:
I tried both of those first (using the getLengthRepresentation, and casting to a string) and they both still read out as decimals🙃

Reply:
But I suppose I could just use a case/switch with hard coded strings if we only ever have two cases and you'd like it to be less fancy?

Reply:
Ah okay gotcha - yeah, not a strong preference since this does seem like it could be a worthwhile util function in general; and I suppose this would support a future case where we decide to add other lengths (like 3/4, etc)

Reply:
WAIT I just realized I did not try a combo of *both of the above approaches- first using getLengthRepresentation and THEN casting to a string!!

Reply:
We have a winner folks😅